spine-c/include/spine/Animation.h

```diff
@@ -60,6 +60,8 @@ typedef struct spAnimation {
 } spAnimation;
 
 spAnimation* spAnimation_create (const char* name, int timelinesCount);
+spAnimation* spAnimation_createWithTimelines (const char* name, float duration,
```

Comment:
Then I'd say keep the function in SkeletonBinary as a special case to increase perf, but don't expose it in Animation.h.